components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -7,26 +7,79 @@ documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
 
 When adding a new metric, the workflow is:
 * Decide on which metric type you want to use.
-* Add a new entry to metrics.yaml.
-* Add code to your project to record into the metric by calling Glean.
+* Add a new entry to `metrics.yaml`.
+* Add code to your project to record into the metric by calling glean.
```

Comment:
Can we linkify this list for easy access? Or add a toplevel TOC?

Reply:
Where would these links point at?  Depending on the event type, we have a bunch of targets for these, but not a single one...

Reply:
Yeah, maybe it's hard to fit this into this list.

I think currently nothing allows me to jump directly to `# Adding new metrics`, `# The `metrics.yaml` file`, `# Choosing a metric type`.
Adding these 3 as a TOC at the start of this doc?

But let's not block on this, happy to talk next week for a follow-up or so?

Reply:
Also: `# Common metric parameters`.